sdk/aqueduct/constants/enums.py

```diff
@@ -214,6 +214,7 @@ class SparkRuntimeType(str, Enum, metaclass=MetaEnum):
 
 class K8sClusterStatusType(str, Enum, metaclass=MetaEnum):
     CREATING = "Creating"
+    UPDATING = "Updating"
```

Comment:
Let's document exactly what this means. Also, can you write a bit at the top of the enum about about what the lifecycle of this cluster status is expected to be? `creating -> active -> terminating -> terminated`. And where does updating fit in?